CONTRIBUTING.rst

```diff
@@ -40,12 +40,12 @@ the environment will bootstrap itself automatically. The steps below are for loc
 #. Install `uv <https://docs.astral.sh/uv/getting-started/installation/>`_:
 
 #. Run ``make install`` to create a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_
-   and install the required development dependencies or run the uv sync command manually:
+   and install the required development dependencies or run the ``uv sync`` command manually:
 
    .. code-block:: shell
-        :caption: Installing the documentation dependencies
+        :caption: Installing the development dependencies
 
-        uv install
+        uv sync --python 3.12
```

Comment:
Selecting a python version isn't really necessary here, and if you do select one, it should be the lowest supported version

Reply:
this was to align with the Makefile here: https://github.com/litestar-org/litestar/blob/bb6aa49deb968da086a4b4d8089b16c4b2aede7e/Makefile#L32

should I change both to use 3.8?

fwiw, `uv sync` won't work its own doesn't work if you have python 3.13 installed. building the docs raises an error because `picologging` (which isn't python 3.13 compatible) isn't installed (it gets [conditionally installed on python<3.13](https://github.com/litestar-org/litestar/blob/518af0dda4e1fea75755147fee2a90dee778e121/pyproject.toml#L99)).

Reply:
Hmm. For `docs` we should probably install `--python=3.12` then and omit a specific version for everything else.

Reply:
done in the latest commit